qutebrowser/browser/pdfjs.py

```diff
@@ -202,15 +203,30 @@ def _read_from_system(system_path, names):
     return (None, None)
 
 
+def _get_pdfjs_basename():
+    """Checks for pdf.js module availability and returns the basename."""
+    final_ext = None
+    exts = ['pdf.js', 'pdf.mjs']
+    for ext in exts:
+        try:
+            final_ext = ext
+            get_pdfjs_res('build/' + ext)
+        except PDFJSNotFound:
+            final_ext = None
+    if not final_ext:
+        raise PDFJSNotFound('build/pdf.js') from None
+    return final_ext
+
+
 def is_available():
```

Comment:
Given that this does something entirely different now (in terms of return value), this should be renamed.

However, we already have `_get_pdfjs_basename` anyways, with the difference that this function also checks for `web/viewer.html`. Wouldn't it be simpler to just leave this one as-is, and call `_get_pdfjs_basename` in `generate_pdfjs_page`?

Reply:
The `is_available` function has been reverted to its original form with a commented check for `pdf.mjs` to retain
some meaningful history.
`_get_pdfjs_basename` now returns either a `str` or `None` given the availability of the mjs-module.
The branching for `no_pdfjs.html` has been adapted to include a testing expression for the function.